src/demos/Banners.vue

```diff
@@ -385,6 +385,43 @@
           </tbody>
         </table>
       </div>
+
+      <div slot="discord">
+        <h2>Discord Banner</h2>
+        <Accordion :openedTitle="'Hide Code'" :closedTitle="'Show Code'">
+          <div slot="content">
+            <pre>
+<code>&lt;Banner-Discord&gt;&lt;/Banner-Discord&gt;</code></pre>
+          </div>
+        </Accordion>
+
+        <Banner-Discord></Banner-Discord>
```

Comment:
Do we have a standard for this?

<banner-discord> vs <Banner-Discord>
Isnt it only single word ie <Button> tags that need to be capitalized?

Reply:
If we do have a standard I was not informed about it.